src/purchases.ts

```diff
@@ -495,11 +510,20 @@ export default class Purchases {
     /**
      * Enables/Disables debugs logs
      * @param {boolean} enabled Enable or not debug logs
+     * @deprecated, use setLogLevel instead
      */
     public static async setDebugLogsEnabled(enabled: boolean): Promise<void> {
         RNPurchases.setDebugLogsEnabled(enabled);
     }
 
+    /**
+     * Used to set the log level. Useful for debugging issues with the lovely team @RevenueCat.
```

Comment:
Might be good to mention what's the default (I believe it's `INFO`)

Reply:
👍🏻